openmdao/components/func_comp_common.py

```diff
@@ -0,0 +1,72 @@
+"""
+Define functions and objects common to the ExplicitFuncComp and ImplicitFuncComp classes.
+"""
+
+try:
```

Comment:
Not sure I understand the jax block. Are users going to import `jax`, `jnp`, etc. from components.fun_comp_common?  Or are they going to import this flag "jax" that tells them whether jax is installed correctly?

Reply:
This block is a leftover and isn't needed any more.  I'll remove it.